src/requirements.txt

```diff
@@ -1,7 +1,7 @@
 Flask==2.3.2
 azure-identity==1.16.1
-azure-appconfiguration-provider==2.0.0b2
-featuremanagement[AzureMonitor]==2.0.0b2
+azure-appconfiguration-provider
```

Comment:
Is there a reason we remove the version, and could a breaking change result in unexpected consequences? Perhaps we can gate against a major version upgrade?

Reply:
Done, is now ~=2.0.0 so the latest none major version will auto install